src/map/script.c

```diff
@@ -2680,6 +2680,13 @@ struct script_data *get_val(struct script_state* st, struct script_data* data) {
 	prefix = name[0];
 	postfix = name[strlen(name) - 1];
 
+	if (strlen(name) > 32) {
+		ShowError("script_get_val: variable name too long. '%s'\n", name);
```

Comment:
not 32, but 31.
Also probably it should be some where in parser if you want prevent using it.
Same for second check